socorro/lib/search_common.py

```diff
@@ -442,8 +441,6 @@ def convert_to_type(value, data_type):
         value = datetimeutil.string_to_datetime(value)
     elif data_type == 'date' and not isinstance(value, datetime.date):
         value = datetimeutil.string_to_datetime(value).date()
-    elif data_type == 'json' and isinstance(value, str):
```

Comment:
👍. Also, as tested, ``date`` is unused:

https://github.com/mozilla-services/socorro/blob/ff6dbb40e1c32d6b2954e19807445e9d99306c6d/socorro/unittest/external/es/test_super_search_fields.py#L266-L267

Reply:
Funny! I want to keep `date`. Bug 1376453 covers pulling the date out of the submission timestamp allowing people to facet on that. That's a P5, but it'd help us since it would simplify some of the goofiness of `fetch_crashids` and some other things. I'll switch that to a P3.